src/theme/SearchBar.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import OriginalSearchBar from '@theme-original/SearchBar'
+import { TraceEvent } from '@uniswap/analytics'
+import { BrowserEvent, EventName } from '@uniswap/analytics-events'
+
+export default function SearchBar(props) {
```

Comment:
naming: `SearchBarWithAnalytics`? Makes it clear on the difference from the original